apps/comments-ui/src/components/content/Comment.tsx

```diff
@@ -298,6 +298,32 @@ const CommentHeader: React.FC<CommentHeaderProps> = ({comment, className = ''})
 
         const element = (e.target as HTMLElement).ownerDocument.getElementById(comment.in_reply_to_id);
         if (element) {
+            const contentElement = element.querySelector('[data-testid="comment-content"]');
```

Comment:
We should avoid using test id's in our actual code. If there are elements that need to be selected I think a custom data property would be better.

Reply:
Removed this with my changes.